### PR TITLE
[WIP] Improve test_odoo_lint script to search the bin path of the rst2html command and used to run the subbprocess pipe to get the result.

### PR DESCRIPTION
**Pull request by zaoral**
Fix Vauxoo/maintainer-quality-tools#30
Task [vx#3932](https://www.vauxoo.com/web#id=3932&view_type=form&model=project.task&action=138) 

NOTE: This PR is WIP yet.

### CODE REVIEW

File: travis/test_odoo_lint [hunk 1]
```diff
@@ -279,13 +280,25 @@ class OdooLint(object):
                     'Legagy mode: Using description in __openerp__.py file')
                 self.desc_legacy.append(module)
 
+    def find_in_path(name):
+        """
+        find the bin path for the given command.
+        """
+        path = os.environ.get('PATH', os.defpath).split(os.pathsep)
+        if config.get('bin_path') and config['bin_path'] != 'None':
```
Comment (moylop260) [inline travis/test_odoo_lint]: I don't see the config variable defined

**Comment by moylop260** [comment]
@zaoral 
FYI I created a new comment

**Comment by zaoral** [comment]
@moylop260 Sorry Moy, this PR is in WIP. I will make the test and let you know when it can be review.
Best Regards

**Comment by moylop260** [comment]
Ok, I got it
I added a `NOTE: This PR is WIP yet.` in first comment.

**Comment by moylop260** [comment]
This PR will fix #30

**Pull request closed without merging**